machine/v1beta1/types_gcpprovider.go

```diff
@@ -134,6 +145,15 @@ type GCPKMSKeyReference struct {
 	Location string `json:"location"`
 }
 
+// GCPAcceleratorConfig describes type and count of accelerator cards attached to the instance on GCP.
+type GCPAcceleratorConfig struct {
+	// AcceleratorCount is number of AcceleratorType accelerators (GPUs) to be attached to an instance
+	AcceleratorCount int64 `json:"acceleratorCount"`
```

Comment:
Using an int64 allows for a lot of accelerator cards!  Probably an int32 would suffice.  

The "Accelerator" prefix seems redundant (`spec.guestAccelerators[*].acceleratorCount`).

Reply:
based on https://pkg.go.dev/google.golang.org/api/compute/v1#AcceleratorConfig

Reply:
> follows https://pkg.go.dev/google.golang.org/api/compute/v1 convention

Can you describe the value this provides to a user of the API?  What is the logical upper bound?

Reply:
The upper bound of this will be in the order of two digits, if not 1, we can definitely get away with an int32

Based on [the limits described in the docs](https://cloud.google.com/compute/docs/gpus#gpus-list), the upper bound is 16 GPUs per instance